app/pibakery.js

```diff
@@ -70,11 +70,12 @@ var tempBlocks = []
   * @return null
 */
 function getOsPath () {
-  if (process.platform == 'darwin') {
+  var platform = process.platform;
+  if ((platform == 'darwin') || (platform == 'linux') || (platform == 'freebsd')) {
     // Mac stores the OS in Application Support
     return path.normalize('/Library/Application Support/PiBakery/os/')
```

Comment:
Of course more edits are needed. This does get PiBakery to boot, I apologise if the title seems misleading.

I've actually made a lib to allow control of such statements, this was a quick hack, I'll close the commit for now as my work at the moment is basically a ground-up rewrite of some components to make dealing with platforms a lot simpler